test/timer.cpp

```diff
@@ -76,7 +76,11 @@ BOOST_AUTO_TEST_CASE(timer_start_stop)
 
     BOOST_REQUIRE_EQUAL(tick_count, required_tick_count);
 
+#ifdef WIN32
+    BOOST_REQUIRE_LE(abs(test_duration - required_duration), milliseconds(16*required_tick_count));
+#else
     BOOST_REQUIRE_LE(abs(test_duration - required_duration), milliseconds(10));
+#endif
```

Comment:
The test should check that the _minimum_ duration after a number of iterations is higher than some lower bound. Especially since I added coalesced timers to Boost.Thread, just because you ask for 10ms has little to do with what you actually get.